packages/cli/src/commands/build.ts

```diff
@@ -35,10 +35,21 @@ export default class Build extends Command {
 
     // Get the output location from the project package.json main field
     const pjson = JSON.parse(readFileSync(path.join(directory, 'package.json')).toString());
+
+    let buildPath = 'src/index.ts';
```

Comment:
read pjson.main for default build path.

Reply:
what is the type of buildPath, it could be a array now, because with this PR we support multiple entries.